### ISSUE TITLE
The function `coll.Index` can not be evaluated

### ISSUE THREAD

**Issue by ProbstDJakob** [hairyhenderson/gomplate#1682]
The function `coll.Index` can not be evaluated and results in a failure. I've tried this with the latest version 3.11.4 and a command copied from the documentation:

```console
$ gomplate -i '{{ coll.Index "foo" (dict "foo" "bar") }}'
20:07:37 ERR  error="failed to render template <arg>: template: <arg>:1:3: executing \"<arg>\" at <coll>: can't evaluate field Index in type interface {}"
```

**Comment by hairyhenderson** [comment]
Hi @ProbstDJakob - sorry about that. That particular function is only in the `main` branch at this point. It will be available in the next version (v4.0.0). Unfortunately the documentation is only currently able to reflect the state of `main`, and I haven't had the time to fix that yet.

**Comment by hairyhenderson** [comment]
Given there's no action necessary here, I'm going to close this. Sorry again for the inconvenience!
